dbms/src/Storages/DeltaMerge/Filter/Unsupported.h

```diff
@@ -37,14 +37,16 @@ class Unsupported : public RSOperator
 
     String name() override { return "unsupported"; }
 
-    Attrs getAttrs() override { return {}; }
+    ColIds getColumnIDs() override { return {}; }
 
     String toDebugString() override
     {
-        return R"({"op":")" + name() + //
-            R"(","reason":")" + reason + //
-            R"(","content":")" + content + //
-            R"(","is_not":")" + DB::toString(is_not) + "\"}";
+        return fmt::format(
+            R"({{"op":"{}","reason":"{}","content":"{}","is_not":"{}"}})",
+            name(),
+            reason,
+            content,
+            DB::toString(is_not));
```

Comment:
```suggestion
            is_not);
```

Reply:
seems `is_not` is not used, can we remove directly?

Reply:
removed